requirements.txt

```diff
@@ -4,3 +4,5 @@ scipy>=1.0.0
 tensorflow==1.3
 tensorflow-tensorboard>=0.1.8
 quantum-blackbird
+toml
```

Comment:
Should these have fixed versions?

Reply:
Yes, probably. We should fix this in a new PR